modules/renter/hostdb/consts.go

```diff
@@ -43,7 +43,7 @@ var (
 	// scanningThreads is the number of threads that will be probing hosts for
 	// their settings and checking for reliability.
 	scanningThreads = build.Select(build.Var{
-		Standard: int(40),
+		Standard: int(20),
```

Comment:
back down to 20 scanning threads. I worry that too many concurrent connections makes scans more likely to fail. Also now that hosts are added to hostdb right away and guaranteed to be scanned, it's not as important to scan hosts quickly.